frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-as-code.feature

```diff
@@ -152,3 +152,64 @@ Feature: Perform Actions on repository
               And user clicks Create GitHub App button in Create GitHub App page
              Then user will be redirected to GitHub App details
               And user will see App Name as "pac-app123", App Link as "https://github.com/apps/pac-app123" and Secret as "pipelines-as-code-secret" in "openshift-pipelines" namespace
+
+
+        @regression @odc-6461 @manual
+        # Manual test case as pipeline 1.8 is not yet available
+        Scenario Outline: Add a Git repository to pipeline as code using webhook url: P-11-TC14
+            Given user has installed OpenShift Pipelines Operator version 1.8 with catalog source "<catalog_yaml>" and image content policy "<image_content_policy_yaml>"
+              And user is at repositories page
+             When user clicks on Create Repository button
+              And user enters Git Repo URL of repository as "<repository_url>"
+              And user enters Name of repository as "<repository_name>"
```

Comment:
This gets auto populated, but users can modify it.

Reply:
Yes so for sake of testing I'm modifying it